### ISSUE TITLE
Wurst Client detection returning NOT_DETECTED in v1.3.1

### ISSUE THREAD

**Issue by taianh20102011** [branduzzo/CheckHacks#11]
### Expected Behavior
CheckHacks should be able to detect players using Wurst Client during a check scan.

### Current Behavior
When performing a check on a player running Wurst Client, it always returns `NOT_DETECTED`. Additionally, some modules occasionally trigger a `PROTECTED` status on the first check.

### Environment
- **Plugin Version:** CheckHacks v1.3.1
- **Server Version:** Paper 1.21.4

### Configuration Snippet
```yaml
  wurst:
    display-name: Wurst
    key: wurst.options.title
    mode: KEYBIND

**Comment by Runkang10** [comment]
Wurst has patched the method used to detect the client. There's no fix for this unfortunately.

**Comment by WindsorMacmillan** [comment]
> Wurst has patched the method used to detect the client. There's no fix for this unfortunately.

Not very soon most hacking clients will be all immune to this detecting method, sadly.

**Comment by Runkang10** [comment]
At some point Mojang might patch it too.
